### PR TITLE
applications: nrf_desktop: Add GPIO leds to nrf52820dongle

### PR DESCRIPTION
**Pull request by pdunaj**
Jira:NCSDK-10299

Signed-off-by: Pawel Dunaj <pawel.dunaj@nordicsemi.no>

### CODE REVIEW

File: applications/nrf_desktop/configuration/nrf52820dongle_nrf52820/dts.overlay [hunk 1]
```diff
@@ -1,3 +1,39 @@
+/ {
+	gpioled1 {
+		compatible = "gpio-leds";
+		status = "okay";
+		label = "LED BLE State";
```
Comment (MarekPieta) [inline applications/nrf_desktop/configuration/nrf52820dongle_nrf52820/dts.overlay:5]: System state

File: applications/nrf_desktop/configuration/nrf52820dongle_nrf52820/dts.overlay [hunk 2]
```diff
@@ -1,3 +1,39 @@
...
+
+		led0_g: led_0 {
+			gpios = <&gpio0 15 GPIO_ACTIVE_LOW>;
+			label = "Green LED 0";
+		};
+	};
+	gpioled2 {
+		compatible = "gpio-leds";
+		status = "okay";
+		label = "LED System State";
```
Comment (MarekPieta) [inline applications/nrf_desktop/configuration/nrf52820dongle_nrf52820/dts.overlay:15]: BLE state

**Review by MarekPieta — APPROVED** [review]
A few small changes

**Comment by pdunaj** [comment]
rebase + CI rerun

File: applications/nrf_desktop/configuration/nrf52820dongle_nrf52820/led_state_def.h [hunk 3]
```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
```
[the remark this answers is not in the document]
Comment (pdunaj) [reply applications/nrf_desktop/configuration/nrf52820dongle_nrf52820/led_state_def.h]: fixed

File: applications/nrf_desktop/configuration/nrf52820dongle_nrf52820/dts.overlay (hunk 1 above)
Comment (pdunaj) [reply applications/nrf_desktop/configuration/nrf52820dongle_nrf52820/dts.overlay]: changed

File: applications/nrf_desktop/configuration/nrf52820dongle_nrf52820/dts.overlay (hunk 2 above)
Comment (pdunaj) [reply applications/nrf_desktop/configuration/nrf52820dongle_nrf52820/dts.overlay]: changed

**Review by emob-nordic — APPROVED** [review]

**Pull request merged**